src/components/board/board.test.js

```diff
@@ -1,8 +1,42 @@
 
+const FILES = ['Jack', 'Seamore', 'Hugh', 'Michael', 'Benjamin', 'Dixie', 'Justin', 'William'];
```

Comment:
Nice - I see what you're doing here - perfect. See my not on the other test file. The makeBoard function should always return an array how you want it - always a A1 or whatever ID you choose. Anything else, it should not return. Does that make sense? So you'll need to update the makeBoard function check the values of the files and ranks being passed in to be something you expect, which is what you're trying to do here using these names. Good idea = perfect - just expand on it and all.